app/controllers/points_controller.rb

```diff
@@ -16,35 +16,21 @@ def index
 
   def new
     @point = Point.new
-    @services = Service.all
     @topics = Topic.all
     @cases = Case.all
-    if @query = params[:service_id]
-      @point['service_id'] = params[:service_id]
-    end
     @service_url = @point.service ? @point.service.url : ''
   end
 
   def create
     @point = Point.new(point_params)
+    @cases = Case.all
+
     @point.user = current_user
-    if params[:has_case]
-      if @point.case.nil? || @point.status.blank? || @point.source.blank?
-        flash[:alert] = "Oops! If you use a case, make sure that all the form fields are filled in before submitting!"
-        render :new
-      elsif
-        @point.update(title: @point.case.title, rating: @point.case.score, analysis: @point.case.description || @point.case.title, topic_id: @point.case.topic_id)
-        if @point.save
-          redirect_to service_path(@point.service)
-          flash[:notice] = "You created a point!"
-        else
-          render :new
-        end
-      end
-    elsif params[:only_create]
+    @point.topic_id = @point.case.topic_id
```

Comment:
If I don't select a case when creating a new point, this leads to

    undefined method `topic_id' for nil:NilClass`

Reply:
This is the wrong version. Are you sure you pulled from origin 201805 case descriptions?

Reply:
Wait, never mind. Will investigate.